### DISCUSSION TITLE
How to upgrade the portable version of Laragon?

### DISCUSSION THREAD

**Discussion by GeraldVeeneman** [leokhoa/laragon#434] [Q&A] [answered]
I have been using the portable version of Laragon 5 and I just wondered  what is the best way to upgrade to version 6. 

Can I just download this [laragon.exe](https://github.com/leokhoa/laragon/releases/download/6.0.0/laragon.exe) file and replace it with the old version? I already tried this way and it seems to work but I'm not sure if this is the right method.

**Comment by Pen-y-Fan** [comment, marked as answer]
Yes, I believe the original forum post suggested replacing the **laragon.exe** with the new version.

Unfortunately, the original post is no longer available.

**Comment by GeraldVeeneman** [comment]
OK, thanks for your reply.

Op zo 12 mrt 2023 om 16:11 schreef Pen y Fan ***@***.***>:
